pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -45,8 +45,8 @@ type InTreeToCSITranslator interface {
 	IsMigratableIntreePluginByName(inTreePluginName string) bool
 	GetInTreePluginNameFromSpec(pv *v1.PersistentVolume, vol *v1.Volume) (string, error)
 	GetCSINameFromInTreeName(pluginName string) (string, error)
-	TranslateInTreePVToCSI(pv *v1.PersistentVolume) (*v1.PersistentVolume, error)
-	TranslateInTreeInlineVolumeToCSI(volume *v1.Volume, podNamespace string) (*v1.PersistentVolume, error)
+	TranslateInTreePVToCSI(logger klog.Logger, pv *v1.PersistentVolume) (*v1.PersistentVolume, error)
+	TranslateInTreeInlineVolumeToCSI(logger klog.Logger, volume *v1.Volume, podNamespace string) (*v1.PersistentVolume, error)
```

Comment:
We don't have any out-of-tree users of this API, right?

Otherwise a `ctx context.Context` parameter would be better, just in case that we ever need it. If it's in-tree, we can still switch later when the need arises.

/lgtm
/approve

For hack config update.

Reply:
Thank you.

> We don't have any out-of-tree users of this API, right?

To the best of my knowledge, there don't seem to be any instances where out-of-tree users, including CSI Sidecar and kube-scheduler-simulator, directly use `TranslateInTreePVToCSI` and `TranslateInTreeInlineVolumeToCSI` .